frontend/components/commons/forms/questions-form/fields/mono-selection/MonoSelectionRenderStyle1.component.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="container">
+    <div class="inputs-area">
+      <div class="input-button" v-for="option in options" :key="option.id">
+        <input
+          type="checkbox"
+          :name="option.text"
+          :id="option.id"
+          v-model="option.value"
+          @change="$emit('on-select', option)"
+        />
+        <label
+          class="label-text cursor-pointer"
+          :class="{ 'label-active': option.value }"
+          :for="option.id"
+          v-text="option.text"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "MonoSelectionRenderComponent",
```

Comment:
Should the name match the file name?

Reply:
there will not be any problem but it's better to match both name, since name is only used for good practice (and debugging with devtools). So I will correct the name.